pkg/reconciler/revision/reconcile_resources.go

```diff
@@ -203,7 +203,7 @@ func hasDeploymentTimedOut(deployment *appsv1.Deployment) bool {
 func (c *Reconciler) reconcileSecret(ctx context.Context, rev *v1.Revision) error {
 	ns := rev.Namespace
 	logger := logging.FromContext(ctx)
-	logger.Info("Reconciling Secret: ", networking.ServingCertName, " at namespace: ", ns)
+	logger.Info("Reconciling Secret for knative-internal-tls: ", networking.ServingCertName, " at namespace: ", ns)
```

Comment:
question: 

should this be `system-internal-tls`, or did you use `knative-internal-tls` on purpose?

Reply:
Ah, no that should be `system-internal-tls`, I'll fix it.